packages/http-client-csharp/generator/Microsoft.Generator.CSharp/test/Providers/ModelProviders/ModelCustomizationTests.cs

```diff
@@ -500,5 +500,99 @@ public async Task DoesNotReplaceDefaultConstructorIfNotCustomized()
             Assert.IsTrue(ctors.Any(c => c.Signature.Modifiers.HasFlag(MethodSignatureModifiers.Public)));
             Assert.IsTrue(ctors.Any(c => c.Signature.Modifiers.HasFlag(MethodSignatureModifiers.Internal)));
         }
+
+        // Validates that if a required literal property is customized, then the default ctor
+        // does not include the custom property as a parameter.
+        [Test]
+        public async Task DoesNotIncludeReqCustomLiteralInDefaultCtor()
+        {
+            var plugin = await MockHelpers.LoadMockPluginAsync(
+                inputModelTypes: [
+                    InputFactory.Model(
+                        "mockInputModel",
+                        usage: InputModelTypeUsage.Input,
+                        properties:
+                        [
+                            InputFactory.Property("Prop1", new InputLiteralType(
```

Comment:
Is there no InputFactory method for literals?

Reply:
thanks, I missed this.